### PR TITLE
Portfolio Project: Cards, About and Form Design 

### PR DESCRIPTION
**Pull request by jlcarrascof**
At the end of this milestone, the work done so far was: 

1. Added  and styled 4 info cards.
2. Created descriptive text in every card.
3. Added a styled button with hover.
4. Created About Myself section.
5. Created social networks section to contact.
6. Created a form following the Figma model.
7. Fixed linters mistakes.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -34,5 +34,296 @@ <h1>I’m Alexandra. <br />Glad to see you!</h1>
...
+                            </li>
+                            <li>
+                                <a href="https://github.com/over-geek"><img src="asset/github.svg" alt="github icon"></a>
+                            </li>
+                            <li>
+                                <a href="#"><img src="asset/thumbs.svg" alt="thumbs icon"></a>
+                            </li>
+                        </ul>
+                    </div>
+                    -->
```
Comment (mahbubul14) [inline index.html]: - [ ] Kindly remove the inline comments from your code. The old pieces of code as inline comments make your project look messy. If you need to review a previous version of your code, you can always use git history. :+1:

File: index.html [hunk 2]
```diff
@@ -34,5 +34,296 @@ <h1>I’m Alexandra. <br />Glad to see you!</h1>
...
+                            <li>javascript</li>
+                        </ul>
+                    </div>
+                    <div>
+                        <button class="btn">See Project</button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
```
Comment (mahbubul14) [inline index.html:168]: - [ ] Kindly try to match your design to the Figma design. The background of the work section is not matched the Figma design. Also, there is a horizontal scrollbar on your design. kindly fix the issue. :+1: 
[horizontal scrollbar.webm](https://user-images.githubusercontent.com/70483835/227731205-4f28679a-886d-493f-b746-bb913ee32d57.webm)

Your design             |  Figma Design
:-------------------------:|:-------------------------:
yourdesign_work | figma_work

File: index.html [hunk 3]
```diff
@@ -34,5 +34,296 @@ <h1>I’m Alexandra. <br />Glad to see you!</h1>
...
+                    <li class="skill-item bottom-bar">
+                        <p>Skills</p>
+                    <button>
+                        <img src="/assets/images/arrow-right.png" alt="arrow icon">
+                    </button>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </section>
```
Comment (mahbubul14) [inline index.html]: - [ ] Please try to match your design to the Figma design. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
yourdesign_myself | figma_myself

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @jlcarrascof,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] In the author section of the readme file, Kindly make those text linkable.

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [ ] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "fixing bug".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 1 above)
Comment (jlcarrascof) [reply index.html]: Thanks a lot, dear @mahbubul14 .... This job is fixed.

File: index.html (hunk 2 above)
Comment (jlcarrascof) [reply index.html:168]: This issue is corrected too

File: index.html (hunk 3 above)
Comment (jlcarrascof) [reply index.html]: And finally and hopefully, this one too.

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @jlcarrascof   👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job on implementing the project requirements! 🥇 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉

## Highlights
✔ Awesome design👍 
✔ Made the required changes👍 
✔ No linter errors👍
### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/enus/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jlcarrascof** [comment]
Thanks you, guys @mahbubul14 and @TuyishimireEric that worked on 2/3 projects this week. Your feedback and suggestions were very valuable as the time you spent on me.

Have a nice weekend,  my buddies and hopefully I'll see you around 🙂🙂🙏🙏🤝🤝

Javier.-

**Pull request merged**